main/go.mod

```diff
@@ -0,0 +1,21 @@
+module github.hpe.com/sharkysharks/go-github-app-boilerplate
```

Comment:
i think this just needs to be `github.com/sharkysharks`. minus the `hpe`

Reply:
oops!